dfx.json

```
@@ -2,12 +2,13 @@
   "version": 1,
   "dfx": "0.8.4",
   "canisters": {
-    "nft": {
-      "build": "cargo build --target wasm32-unknown-unknown --release --package nft",
-      "candid": "nft/candid/nft.did",
-      "wasm": "target/wasm32-unknown-unknown/release/nft.wasm",
-      "type": "custom"
-    }
+    "canisters": {
+      "nft": {
+    	"package": "nft",
+    	"candid": "nft/candid/nft.did",
+    	"type": "rust"
+      }
+    },
```

Mind fix the following, as we have nested "canisters" for field "canisters"?

```suggestion
      "nft": {
    	"package": "nft",
    	"candid": "nft/candid/nft.did",
    	"type": "rust"
      },
```

Ref:
https://smartcontracts.org/docs/release-notes/0.8.4-rn.html